cs/src/Contracts/RateStatus.cs

```diff
@@ -27,4 +27,47 @@ public class RateStatus : ResourceStatus
     /// </summary>
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public long? ResetTime { get; set; }
+
+    /// <inheritdoc/>
+    public override string ToString()
+    {
+        var count = base.ToString();
+        if (PeriodSeconds == null)
+        {
+            return count;
+        }
+
+        if (PeriodSeconds.Value == 1)
+        {
+            return count + "/s";
+        }
+        else if (PeriodSeconds.Value == 60)
```

Comment:
nit: all these `else` are not necessary because there is `return` inside `if` branches.

Reply:
Sure, but I like the `else` keywords anyway because the clauses are all related. And without `else` the style rules would force an extra blank line after each `}`.